CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# 1.12.0
+# Added
+- Add stateproof to keyreg transaction (#278)
+- Create response object for "AtomicTransactionComposer.Execute" (#276)
+- Support ABI reference types and other improvements (#273)
+- Add CreateDryrun function (#265)
+- Add EncodeAddress (264)
```

Comment:
I don't think this is a big deal, but I just noticed this one doesn't have a pound sign in front of the issue number.
```suggestion
- Add EncodeAddress (#264)
```

Reply:
Oh, yes, please add that, as it will turn it into a link automatically.